src/platform-includes/performance/span-api-version/javascript.mdx

```diff
@@ -0,0 +1,5 @@
+<Note>
+
+The span APIs require SDK version `7.65.0` or higher. If you are using an older version of the SDK, you can use the [explicit transaction APIs](#start-transaction) for custom instrumentation.
```

Comment:
l: Is there a reason why this note is different to the Node version? I'd prefer the Node version as it explicitly names the APIs.

Reply:
nope, I just forgot lol

good call!